README.md

```diff
@@ -1,65 +1,75 @@
+---
+page_type: sample
+languages:
+- javascript
+products:
+- azure
+description: "This QuickStart shows how to store a secret in Key Vault and how to retrieve it using a Web app. This web app may be  run locally or in Azure."
+urlFragment: key-vault-node-quickstart
+---
 
 # Quickstart: Set and retrieve a secret from Azure Key Vault using a Node Web App 
 
-This QuickStart shows how to store a secret in Key Vault and how to retrieve it using a Web app. This web app may be  run locally or in Azure. The quickstart uses Node.js and Managed service identities (MSIs)
+This Quickstart shows how to store a secret in Key Vault and how to retrieve it using a Web app. This web app may be run locally or in Azure. The Quickstart uses Node.js and Managed Service Identities (MSIs)
 
 > * Create a Key Vault.
 > * Store a secret in Key Vault.
 > * Retrieve a secret from Key Vault.
 > * Create an Azure Web Application.
-> * [Enable managed service identities](https://docs.microsoft.com/azure/active-directory/managed-service-identity/overview).
+> * [Enable Managed Service Identities](https://docs.microsoft.com/azure/active-directory/managed-service-identity/overview).
 > * Grant the required permissions for the web application to read data from Key vault.
 
-Before you proceed make sure that you are familiar with the [basic concepts](key-vault-whatis.md#basic-concepts).
+Before you proceed make sure that you are familiar with the [basic concepts](https://docs.microsoft.com/en-us/azure/key-vault/key-vault-overview).
 
 ## Prerequisites
 
-* [Node JS](https://nodejs.org/en/)
+* [Node.js](https://nodejs.org)
 * [Git](https://www.git-scm.com/)
 * [Azure CLI](https://docs.microsoft.com/cli/azure/install-azure-cli?view=azure-cli-latest) 2.0.4 or later
 * An Azure subscription. If you don't have an Azure subscription, create a [free account](https://azure.microsoft.com/free/?WT.mc_id=A261C142F) before you begin.
 
-## Login to Azure
+## Log in to Azure
 
-To log in to Azure using the CLI, you can type:
+1. Open a command prompt, i.e. cmd, terminal, etc
+2. Execute the following command to log in to Azure
+`az login`
 
 ```azurecli
 az login
 ```
 
-## Create resource group
+## Create Resource Group
 
-Create a resource group with the [az group create](/cli/azure/group#az-group-create) command. An Azure resource group is a logical container into which Azure resources are deployed and managed.
+Create a Resource Group with the [az group create](https://docs.microsoft.com/en-us/azure/azure-resource-manager/manage-resources-cli) command. An Azure Resource Group is a logical container into which Azure resources are deployed and managed.
+
+When you create a Resource Group you have give it a unique custom name. Please think of a custom name for your resource group and replace the text below "YourResourceGroupName" with the custom name you created.
```

Comment:
You'll want to include the "<" and ">" here to make sure they know to replace those as well.